### PR TITLE
[AIRFLOW-XXX] Add logo info to readme

### PR DESCRIPTION
**Pull request by leahecole**
I added info about logo usage to the README! One thing I wasn't sure is if line 48 needs to be included in the README - I autogenerated it when I built the docs locally, but if I should remove it because it will be generated as part of the build, let me know. 

If needed, I can also add a README to the docs/img/logos folder - I didn't do that yet because I was unsure how it would affect other generated docs

### CODE REVIEW

**Comment by ashb** [comment]
> ne thing I wasn't sure is if line 48 needs to be included in the README - I autogenerated it when I built the docs locally, but if I should remove it because it will be generated as part of the build, let me know

Yes, keep it there. We don't have any other autobuilding of the readme than what we have from the pre-commit hooks.

File: README.md [hunk 1]
```diff
@@ -441,6 +442,10 @@ are responsible for reviewing and merging PRs as well as steering conversation a
 If you would like to become a maintainer, please review the Apache Airflow
 [committer requirements](https://cwiki.apache.org/confluence/display/AIRFLOW/Committers).
 
+## Can I use the Apache Airflow logo in my presentation?
+
+Yes! Be sure to abide by the Apache Foundation [trademark policies](https://www.apache.org/foundation/marks/pmcs#graphics). The most up to date logos are found in [the repo](/docs/img/logos)
```
Comment (ashb) [inline README.md]: This may not be the best link as that page is mostly focused at the project itself. Let me see if I can find a better one.

Comment (ashb) [reply README.md]: Something on https://www.apache.org/foundation/marks/ perhaps.

Comment (ashb) [reply README.md]: https://www.apache.org/foundation/marks/#books more specifically?

Comment (leahecole) [reply README.md]: How about this one? http://www.apache.org/foundation/marks/guide#naming

Comment (leahecole) [reply README.md]: Oh perfect love the #books one. I'll make that change now.

**Review by ashb — APPROVED** [review]

**Comment by mik-laj** [comment]
We also have brandbook: https://cwiki.apache.org/confluence/display/AIRFLOW/Brandbook

**Comment by leahecole** [comment]
I can add a link to that in this PR also, or would it be better to open a separate PR @mik-laj? 

> We also have brandbook: https://cwiki.apache.org/confluence/display/AIRFLOW/Brandbook

**Comment by mik-laj** [comment]
In this PR it is okay for me

**Comment by leahecole** [comment]
Great. Added it!

**Comment by leahecole** [comment]
rebased to newest master because of Jarek's email

**Comment by potiuk** [comment]
Thanks @leahecole !

**Pull request merged**